build/Makefile

```diff
@@ -124,7 +124,7 @@ ifdef NO_RACE_DETECTOR
   RACE_DETECTOR_ARGS=
 endif
 
-go_test_cmd = go test -mod=vendor $(RACE_DETECTOR_ARGS)
+go_test_cmd = go test -mod=vendor $(RACE_DETECTOR_ARGS) -v -run TestPingHTTP
```

Comment:
Was this left in by accident? Doesn't this limit what tests are run?

Reply:
Resolved, sorry, was fixing this test which is failing on `make kind-test-e2e`